voter-dapp/src/containers/VoteData.js

```diff
@@ -87,24 +87,29 @@ function useVoteData() {
           }
         }
 
+        // Data on unique users:
+        const uniqueCommits = Object.keys(uniqueVotersCommitted).length;
+        const uniqueReveals = Object.keys(uniqueVotersRevealed).length;
+        const uniqueClaimers = Object.keys(uniqueRewardClaimers).length;
+        const uniqueRevealsPctOfCommits = uniqueCommits > 0 ? (100 * uniqueReveals) / uniqueCommits : 0;
+        const uniqueClaimersPctOfReveals = uniqueReveals > 0 ? (100 * uniqueClaimers) / uniqueReveals : 0;
+
         // Insert round data into new object.
         newVoteData[newRoundKey] = {
           totalSupplyAtSnapshot: dataForRequest.totalSupplyAtSnapshot,
```

Comment:
This could be `null` if `totalSupplyAtSnapshot` doesn't exit yet, such as in the commit phase, but I can set its default to 0 if that's preferable. But "total supply = 0" seems stranger/less correct than "total supply = N/A"